sdk/typescript/src/types/events.ts

```diff
@@ -12,7 +12,7 @@ export type MoveEvent = {
     transactionModule: string;
     sender: SuiAddress;
     type: string;
-    fields: { [key: string]: any; };
+    fields?: { [key: string]: any; };
```

Comment:
`fields` should always have been optional, i think. Some transactions turned out to have events without fields, this change unblocked the type guard validation